views/users/new.tpl

```diff
@@ -0,0 +1,8 @@
+{{ if .flash }}
+  {{ .flash }}
+{{ end }}
+<form class="" action="/users" method="post">
+  {{ .Form | renderform }}
```

Comment:
Should the password and password confirmation field type password?
![Screen Shot 2564-01-19 at 14 13 13](https://user-images.githubusercontent.com/29707647/105000182-9b12e880-5a60-11eb-8f74-15eed1cebc56.png)

Reply:
@gutakk Yes, I'll fix it on the frontend ticket #23 🙌🏼